vue3-apollo-quasar/src/components/RepoAbout/HomepageUrl.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div v-if="homepageUrl" class="linkContainer">
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      class="linkIcon"
+      fill="none"
+      viewBox="0 0 24 24"
+      stroke="currentColor"
+      stroke-width="2"
+    >
+      <path
+        stroke-linecap="round"
+        stroke-linejoin="round"
+        d="M13.828 10.172a4 4 0 00-5.656 0l-4 4a4 4 0 105.656 5.656l1.102-1.101m-.758-4.899a4 4 0 005.656 0l4-4a4 4 0 00-5.656-5.656l-1.1 1.1"
+      />
+    </svg>
+    <a href="{{homepageUrl}}" class="link" target="_blank">
+      {{ homepageUrl }}
+    </a>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'HomepageUrl',
+  props: {
+    homepageUrl: String,
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+.linkContainer {
+  max-width: 100%;
+  overflow: hidden;
```

Comment:
This could use quasar classes instead:
- [Width](https://quasar.dev/style/other-helper-classes#size-related)
- [Overflow](https://quasar.dev/style/other-helper-classes#scroll-related)

Check out the docs for the various options you have available